apiserver/src/api/mod.rs

```diff
@@ -130,6 +131,18 @@ pub async fn run_server<T: 'static + BottlerocketShadowClient>(
     );
 
     // Set up the actix server.
+    let server_addr;
+
+    // Use IP for KUBERNETES_SERVICE_HOST to decide the IP family for the cluster
+    let k8s_service_addr = env::var("KUBERNETES_SERVICE_HOST").context(error::MissEnvVar)?;
+    if k8s_service_addr.contains(":") {
```

Comment:
Nit: I'd like to see more description about what the formate would be for `k8s_service_addr` in IPV6 and IPV4. Can  you put some comments here to tell why you use `:` to differentiate IPV4 and IPV6.

Reply:
A normal IPv6 address using `:`, and a normal IPv4 address using `.`

Reply:
A normal IPv6 address uses `:` to divide, and a normal IPv4 address uses `.` to divide.